components/DataCard/DataCard.tsx

```diff
@@ -0,0 +1,93 @@
+import { cnb } from 'cnbuilder';
+import { AnimateInView, type AnimationType } from '@/components/Animate';
+import { NumberCounter } from '@/components/NumberCounter';
+import { Container } from '@/components/Container';
+import { FlexBox } from '@/components/FlexBox';
+import { Heading, type HeadingType } from '../Typography';
+import { accentBorderColors, type AccentBorderColorType, type PaddingType } from '@/utilities/datasource';
+import { splitNumberString } from '@/utilities/splitNumberString';
+import * as styles from './DataCard.styles';
+
+export type DataCardProps = React.HTMLAttributes<HTMLDivElement> & {
+  heading?: string;
+  headingLevel?: HeadingType;
+  isDarkTheme?: boolean;
+  barColor?: AccentBorderColorType;
+  body?: React.ReactNode;
+  paddingTop?: PaddingType;
+  cta?: React.ReactNode;
+  isCounter?: boolean;
+  // In number of seconds
+  counterDuration?: number;
+  animation?: AnimationType;
+  delay?: number;
+};
+
+export const DataCard = ({
+  heading,
+  headingLevel = 'h3',
+  barColor,
+  body,
+  cta,
+  paddingTop,
+  isDarkTheme,
+  isCounter,
+  counterDuration,
+  animation = 'slideUp',
+  delay,
+  children,
+  className,
+  ...props
+}: DataCardProps) => {
+  const headingProcessed = isCounter ? splitNumberString(heading) : undefined;
+
+  return (
+    <AnimateInView animation={animation} delay={delay} className={styles.animateWrapper}>
+      <Container
+        as="article"
+        width="full"
+        pt={paddingTop}
+        className={styles.root}
+        {...props}
+      >
+        <FlexBox direction="col" className={styles.flex}>
+          {/* If number counter is enabled, aria-hidden the animated heading and add a SR only heading */}
+          {isCounter && heading && (
+            <Heading as={headingLevel} srOnly>{heading}</Heading>
+          )}
```

Comment:
If there is a number counter in the heading, add a SR only heading with all static heading, then aria-hidden the visible heading with the number counter.